lib/internal/fs/promises.js

```diff
@@ -808,11 +808,22 @@ async function readFile(path, options) {
   return handleFdClose(readFileHandle(fd, options), fd.close);
 }
 
+async function exists(path) {
+  validateString(path, 'path');
```

Comment:
This should be using `getValidatedPath` here to support all kinds of paths.
```suggestion
  path = getValidatedPath(path, 'path');
```

Reply:
Fixed. thanks